pypesto/ensemble/ensemble.py

```diff
@@ -1253,3 +1269,65 @@ def calculate_cutoff(
 
     range = chi2.ppf(q=percentile / 100, df=df)
     return fval_opt + range
+
+
+def calculate_hpd(
+    result: Result,
+    burn_in: int,
+    ci_level: float = 95,
```

Comment:
```suggestion
    ci_level: float = .95,
```
I guess? otherwise 1324 would most certainly throw an error i think (out of bounds)